docs/attributes-registry/thread.md

```diff
@@ -10,6 +10,6 @@ These attributes may be used for any operation to store information about a thre
 <!-- semconv registry.thread(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  |
 |---|---|---|---|
-| `thread.id` | int | Current "managed" thread ID (as opposed to OS thread ID). | `42` |
+| `thread.id` | string | Current "managed" thread ID (as opposed to OS thread ID). | `42` |
```

Comment:
There should be changes in yaml file as well, probably you forgot them to commit? @pyohannes

Reply:
I committed those changes now.